core/api/test/bats/gql/ln-invoice-payment-send.gql

```diff
@@ -4,5 +4,13 @@ mutation lnInvoicePaymentSend($input: LnInvoicePaymentInput!) {
       message
     }
     status
+    transaction {
+      initiationVia {
+        ... on InitiationViaLn {
+          paymentHash
+          paymentRequest
```

Comment:
we should also get the pre-image?

Reply:
(unrelated to this PR)

Reply:
not sure if it is working but we already have it https://github.com/GaloyMoney/galoy/blob/main/quickstart/graphql/public/schema.graphql#L1217